lib/handlers/events/delivery/goals/RespondOnGoalCompletion.ts

```diff
@@ -27,19 +27,21 @@ import { HandleEvent } from "@atomist/automation-client/lib/HandleEvent";
 import {
     addressChannelsFor,
     CredentialsResolver,
-    fetchGoalsForCommit,
+    fetchGoalsFromPush,
     GoalCompletionListener,
     GoalCompletionListenerInvocation,
     RepoRefResolver,
     SdmGoalEvent,
 } from "@atomist/sdm";
+import * as _ from "lodash";
```

Comment:
It doesn't look like any use of this was added.

Reply:
How did that end up in there. Fixing.